install/kubernetes/helm/istio/templates/sidecar-injector-configmap.yaml

```diff
@@ -141,6 +141,9 @@ data:
         - --applicationPorts
         - {{ "\"[[ annotation .ObjectMeta `readiness.status.sidecar.istio.io/applicationPorts` (applicationPorts .Spec.Containers) ]]\"" }}
       {{ "[[- end ]]" }}
+      {{- if .Values.global.trustDomain }}
```

Comment:
can you make this as Values.citadel.trustDomain now that we have subcharts, etc. ?:

Reply:
We moved it to the global context, because we need to read the value from several places (e.g pilot, pilot-agent,...)

Reply:
And it has nothing to do with citadel ( other CAs may be used to generate certs, citadel is just one implementation)